src/test/java/org/jenkinsci/plugins/workflow/log/LogStorageTestBase.java

```diff
@@ -93,6 +93,7 @@
         step1.getLogger().println("one #3");
         overall.getLogger().println("pausing");
         overallHtmlPos = assertOverallLog(overallHtmlPos, "<span class=\"pipeline-node-1\">one #2\none #3\n</span>pausing\n", true);
+        // TODO if we produce output from the middle of a step, we need new span blocks
```

Comment:
An unrelated bug I noticed: the **hide** links do not work reliably for running builds.

Reply:
As an aside: I checked actual `progressiveHtml` HTTP responses and they look sane, and I certainly verified this scenario when developing the feature, so I am not sure why it would not be working now. Possibly a client-side CSS issue. Will look into it separately.